circle.yml

```diff
@@ -0,0 +1,9 @@
+machine:
+  node:
+    version: 0.10.0
```

Comment:
Well that stinks. Looking at their site they just use NVM to manage installing Node.JS. So you could feasibly just have a script which switches to a node version via NVM then runs the tests then switches to the next and runs the tests.

```bash
test:
  override:
    - nvm install 0.10 && npm test && nvm install 0.12 && npm test && nvm install 4.0 && npm test
```

I would think the above would do it (though it is a bit hacky).